devices/stm32f103.yaml

```diff
@@ -72,6 +72,7 @@ _include:
  - ../peripherals/gpio/gpio_f1.yaml
  - ../peripherals/spi/spi_v1_without_FRF_FRE.yaml
  - common_patches/tim/tim_o24ce.yaml
+ - common_patches/tim/tim10_missing_opm.yaml
```

Comment:
```suggestion
```

In RM0009, TIM10/11/13/14 don't have OPM.

Reply:
RM0008 Rev 21 now has OPM bits